posts/intjnotinfp/2020-08-26-first-post.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: post
+author: <intjnotinfp>
```

Comment:
The `<>` brackets aren't needed. No worries- using brackets in instructions  to indicate a 'variable' should go in there is an idiom you'll become more familiar with over time.